larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -19,7 +19,7 @@
 namespace lar_pandora {
 
   void
-  LArPandoraGeometry::LoadDetectorGaps(LArDetectorGapList& listOfGaps)
+  LArPandoraGeometry::LoadDetectorGaps(LArDetectorGapList& listOfGaps, bool m_useActiveBoundingBox)
```

Comment:
Suggest `const bool useActiveBoundingBox`, so it's const and doesn't imply that it's a member variable of LArPandoraGeometry via the prefix `m_`

As a note: We could potentially create a settings class for LArPandoraGeometry, following approach for LArPandoraInput (with LArPandoraGeometry instance having a settings member variable), but this additional argument seems fine for now, as a first foray into passing in new configuration.